servers/src/main/java/tachyon/worker/nio/NIODataServer.java

```diff
@@ -174,7 +175,7 @@ public boolean isClosed() {
     return mShutdownComplete;
   }
 
-  private void read(SelectionKey key) throws IOException {
+  private void read(SelectionKey key) throws Exception {
```

Comment:
after replacing IOException in worker package, a lot more exceptions might be thrown here, but in https://github.com/amplab/tachyon/pull/1110/files#diff-cd657b2cddaa644ef777f43f6e46a1aaR268 and https://github.com/amplab/tachyon/pull/1110/files#diff-cd657b2cddaa644ef777f43f6e46a1aaL272, it is only cared whether exception has been thrown, so I put an Exception here.